ccpp/suites/suite_FV3_RRFS_v1alpha.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_RRFS_v1alpha" lib="ccppphys" ver="1">
```

Comment:
This line is not correct. Please replace `lib="ccppphys" ver="1"` with just `version="1"` - see other SDFs.